internal/utils/document_utils.go

```diff
@@ -5,17 +5,33 @@ import (
 	"github.com/ocp-docs-api/internal/models/document"
 )
 
-func SplitDocumentSlice(source []document.Document, chunkSize uint) ([][]document.Document, error) {
-	if source == nil || chunkSize == 0 {
-		return nil, errors.New("inpit slice is nil or chunkSize is invalid")
+var (
+	inputSliceIsNil = errors.New("inpit slice is nil")
+	chinkSizeIsInvalid = errors.New("chunkSize is invalid")
+	keyAlreadyExistInAMap = errors.New("key already exist in a map")
```

Comment:
обычно линтер заворачивает такую нотацию для ошибок. должно быть 
```suggestion
	errorKeyAlreadyExistInAMap = errors.New("key already exist in a map")
```